scss/_reboot.scss

```diff
@@ -727,3 +727,26 @@ progress {
 [hidden] {
   display: none !important;
 }
+
+// Boosted mod
```

Comment:
Doesn't know if this declaration more need to be in `_root.sccs`?

Reply:
After a try, it works too, the only limit I can see so far (and I'm not sure that it is actually one) is the future relation with :host as described in #1022, what do you think about it ? Are they suitable ? Thanks a lot for your time spent on the review !